conf/messages

```diff
@@ -32,12 +32,14 @@ overview.agent.task1.caption = Complete the sections that apply to your client.
 overview.individual.task1.caption = Complete the sections that apply to you.
 
 overview.agent.task2.heading = View Tax calculation to date
-overview.agent.task2.caption = Provide at least one update before you can view your client’s estimate.
+overview.agent.task2.caption.noUpdate = Provide at least one update before you can view your client’s estimate.
+overview.agent.task2.caption.update = Get Estimation
```

Comment:
From looking at the prototype this should be 'View Estimation'
<img width="876" alt="Screenshot 2021-01-11 at 15 08 24" src="https://user-images.githubusercontent.com/60218795/104199230-1819de80-541f-11eb-84b7-b95d02a8fcbf.png">